integration/spec_helper.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+require_relative "supports/logstash_test_helpers"
+require "fileutils"
+
+# Configure the test environment
+source = File.expand_path(File.join(File.dirname(__FILE__), ".."))
+integration_path = File.join(source, "integration_run")
+
+puts "[integration_spec] configure environment"
+
+if Dir.exists?(integration_path)
```

Comment:
Not sure I understand here.

Since the `integration_run` should be the same as the root directory because of the rsync.
If you run the test again you they hopefully will get the same results, no?

I have added the check on the `Dir.exist?` to allow people to run their tests with `bin/rspec file:number`, which is more convenient and a lot faster to run a portion of the suite.

I agree that this may be a bit of magic, but I think it give flexibility.